real_check/real_check.ps1

```diff
@@ -0,0 +1,57 @@
+#Copyright (c) Microsoft. All rights reserved.
+#Licensed under the MIT license. See LICENSE file in the project root for full license information.
+function Check-Symbols {
```

Comment:
>ction Check-Symbols [](http://example.com/codeflow?start=3&length=19)

I think we should have some tests.
Like we have for the rest of the code.

So I would write the following tests:
- Execute the tool for real_check_test and see that it succeeds
- Execute the tool for real_check_test_reals and see it succeeds
- Execute the tool for both and see it fails

These tests should be automated (a ps1 also) that we should run with cmake (you can add a test to the cmake list of tests and the command for that test be the invocation of powershell with the tests that you add).

Reply:
Adding test in next iteration.